cpp/CMakeLists.txt

```diff
@@ -253,8 +253,8 @@ add_library(cudf
     src/interop/dlpack.cpp
     src/interop/from_arrow.cu
     src/interop/to_arrow.cu
-    src/io/avro/avro.cpp
     src/io/avro/avro_gpu.cu
+    src/io/avro/avro.cpp
```

Comment:
This shuffling of order doesn't look needed as it was previously in alphabetical order

Reply:
yeah, I probably just selected a bunch of lines and hit a sorting shortcut. reverted.